settings.yml

```diff
@@ -18,7 +18,7 @@ show_on:
     - top_menu
     - homepage
     - all
-  description: top_menu refers to the <a href="/admin/site_settings/category/all_results?filter=top_menu">top menu site setting</a> </br> when set to all, enable	&ldquo;above main container&rdquo; below.
+  description: top_menu refers to pages set in the <a href="/admin/site_settings/category/all_results?filter=top_menu">top menu site setting</a>
```

Comment:
btw. it was mentioned in another PR recently - a link like this might not work in subfolder setups?